concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -71,7 +71,9 @@ EOF
 generate_manifest_file() {
   # This exists because concourse does not support boolean value interpolation by design
   enable_auto_deploy=$([ "${ENABLE_AUTO_DEPLOY:-}" ] && echo "true" || echo "false")
+  [ "${DISABLE_CONTINUOUS_SMOKE_TESTS}" = "true" ] && continuous_smoke_tests_trigger="false" || continuous_smoke_tests_trigger="true"
```

Comment:
Nitpick: could you use the same style as the preceding line so that it's easier to read this function? (I *think* the following logic is correct)
```sh
continuous_smoke_tests_trigger=$([ "${DISABLE_CONTINUOUS_SMOKE_TESTS:-}" ] && echo "false" || echo "true")
```